hummingbot/templates/conf_global_TEMPLATE.yml

```diff
@@ -28,6 +28,9 @@ bittrex_secret_key: null
 blocktane_api_key: null
 blocktane_api_secret: null
 
+bybit_api_key: null
+bybit_api_secret: null
```

Comment:
When changing a `*TEMPLATE.yml` file it is mandatory to also update its version number (first element in the same file)